src/main/ui/app/route/About.jsx

```diff
@@ -9,7 +9,7 @@ export default class About extends Component {
                     <h3>About</h3>
 
                     <p>
-                        <Link to="/indexingStatus">Indexing Status</Link>
+                        <Link className="btn btn-lg btn-primary col-md-1 col-sm-12 col-xs-12 buttonfix" to="/indexingStatus">Indexing Status</Link>
```

Comment:
Don't make this a button, just leave it as a plain link. This is advanced functionality that's kind of only there for admin troubleshooting, so I don't want to draw clicks to it.